src/_pytest/fixtures.py

```diff
@@ -1113,6 +1113,9 @@ def fixture(
                 ``fixture_<fixturename>`` and then use
                 ``@pytest.fixture(name='<fixturename>')``.
     """
+    if params is not None and not isinstance(params, (list, tuple)):
```

Comment:
I think we can always create a list, for consistency:

```suggestion
    if params is not None:
```

It also ensures our params list is not affected by changes from the outside. I don't expect this to make any difference performance-wise.